include/flang/Lower/PFTBuilder.h

```diff
@@ -104,62 +82,45 @@ using ConstructStmts = std::tuple<
     parser::MaskedElsewhereStmt, parser::ElsewhereStmt, parser::EndWhereStmt,
     parser::ForallConstructStmt, parser::EndForallStmt>;
 
+using Constructs =
+    std::tuple<parser::AssociateConstruct, parser::BlockConstruct,
+               parser::CaseConstruct, parser::ChangeTeamConstruct,
+               parser::CriticalConstruct, parser::DoConstruct,
+               parser::IfConstruct, parser::SelectRankConstruct,
+               parser::SelectTypeConstruct, parser::WhereConstruct,
+               parser::ForallConstruct, parser::CompilerDirective,
+               parser::OpenMPConstruct, parser::OmpEndLoopDirective>;
+
 template <typename A>
 static constexpr bool isActionStmt{common::HasMember<A, ActionStmts>};
 
 template <typename A>
-static constexpr bool isConstruct{common::HasMember<A, Constructs>};
+static constexpr bool isOtherStmt{common::HasMember<A, OtherStmts>};
 
 template <typename A>
 static constexpr bool isConstructStmt{common::HasMember<A, ConstructStmts>};
 
 template <typename A>
-static constexpr bool isOtherStmt{common::HasMember<A, OtherStmts>};
-
-template <typename A>
-static constexpr bool isGenerated{std::is_same_v<A, CGJump>};
+static constexpr bool isConstruct{common::HasMember<A, Constructs>};
```

Comment:
Yes.  I prefer to keep an order for code where possible.  Alphabetical order is a good fallback that I use quite often, but in this case I prefer declaration order.  Constructs are the odd case, and seem to me to be best placed last everywhere there isn't a more compelling reason to put them somewhere else.

Reply:
Declaration of what order? I'm not following.

Reply:
using ActionStmts = ...
using OtherStmts = ...
using ConstructStmts = ...
using Constructs = ...

(This is what is there now; I may have changed this.  Since constructs are composed of the earlier cases, this is what I think makes sense.)

Code should generally then be placed in this order (and probably in order of subdeclaration in each case) when there is no other more compelling choice.

There could be exceptions, or we could just live with semi-random code order.  I usually prefer to at least attempt some reasonable order to help readers find code.

Reply:
Ok. I follow that. :)